.github/workflows/cicd.yml

```diff
@@ -25,26 +25,6 @@ jobs:
           name: nix-archives
           path: ${{ steps.build-nix-archives.outputs.result }}/
 
-  build-with-cachix:
```

Comment:
Ah, check. I felt a little brazen adding in this change, but I have a similar issue with the examples CI job: it doesn't really test the current build. Didn't touch the examples job though, because it wasn't in the way. 😛